dev-python/nilearn/nilearn-0.1.4.1.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_7 python3_4 )
+
+inherit distutils-r1
+
+DESCRIPTION="Fast and easy statistical learning on NeuroImaging data"
+HOMEPAGE="http://nilearn.github.io/"
+SRC_URI="mirror://pypi/n/${PN}/${PN}-0.1.4.post1.tar.gz -> ${P}.tar.gz"
```

Comment:
You should use ${PV} here or add a ${MY_PY} using regex or the versionator eclass to avoid hardcoding the version number.